net.sourceforge.efaxgtk.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: net.sourceforge.efaxgtk
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: efax-gtk
+rename-desktop-file: efax-gtk.desktop
+rename-icon: efax-gtk
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=session-bus
+  - --share=network
+  - --filesystem=host
```

Comment:
so you need the whole filesystem access?

Reply:
Um, probably not... likely just documents?  Let me look into that...

Reply:
I think it'll at least need `home` (the config file is `~/.efaxgtkrc`)

Reply:
you should use `--persist=.efaxgtkrc` for that, and `--filesystem=xdg-documents` for the docments

Reply:
I'm trying this:

```patch
diff --git a/net.sourceforge.efaxgtk.yaml b/net.sourceforge.efaxgtk.yaml
index 7fc0c0d..59d10c7 100644
--- a/net.sourceforge.efaxgtk.yaml
+++ b/net.sourceforge.efaxgtk.yaml
@@ -10,8 +10,9 @@ finish-args:
   - --socket=x11
   - --socket=session-bus
   - --share=network
-  # Looks like the app uses ~/.efax-gtkrc
-  - --filesystem=home
+  - --persist=.efax-gtkrc
+  - --persist=.efax-gtk_mainwin_save
+  - --filesystem=xdg-documents
   # For file browse dialogs
   - --filesystem=xdg-run/gvfsd
   # For modems

```

And I'm getting:

```
Can't open file/home/wberrier/.efax-gtkrc
```

Not quite sure why that's not working... plus, I think it's nice if the user has a pre-existing configuration that it gets picked up with `home`.